packages/flutter/pubspec.yaml

```diff
@@ -22,17 +22,16 @@ dev_dependencies:
   flutter_goldens:
     sdk: flutter
   fake_async: 1.3.1
-  leak_tracker: 9.0.8
-  leak_tracker_flutter_testing: 1.0.6
+  leak_tracker: 9.0.11
+  leak_tracker_testing: 1.0.5
+  leak_tracker_flutter_testing: 1.0.9
```

Comment:
Do we really want to have runtime dependencies on these packages in package/flutter?

Reply:
First, they are just dev_dependencies.
Second, they will go away after we clean up `testWidgetsWithLeakTracking` from package/flutter/test.

Reply:
Ahh I see now, missed this the way the GH diff was collapsed.